python-checks/src/test/resources/checks/useOfOverrideOrOverloadDecorator.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any
+
+class Class(UnknownParent):
+    @overload
+    def addItem(self, data: Any = ...) -> None:
+        ...
+
+    @override
+    def addItem(self, data: Any = ...) -> None:
+        ...
+
+from reexport_typing_overload_override import reexported_override, reexported_overload
+
+class ClassWithReExports(UnknownParent):
```

Comment:
I think it would be good to express that these are false positive. Possibly through the name of the class, or through some comment.
Also, if you created some ticket to handle this case, you could consider having a small comment to reference it.